deps/source_package/debian/changelog

```diff
@@ -0,0 +1,4570 @@
+edubuntu-meta (15.12.5) xenial; urgency=medium
```

Comment:
We can keep only this year's changelog

Reply:
Yes. I am keeping some lines of `changelog`

Reply:
I don't think we need any previous logs, since this is not a continuation of ubuntu-meta.
Just the initial log of Deepanshu is enough (and it should mention debian-edu instead of ubuntu-edu).

Anyway, since this is just an internal package, that has no chance to be published on DebianEdu, it doesn't matter much what is written on the log. After this project, maybe this package will not be maintained. It served the purpose of helping us to learn how to build deb packages, and in my opinion this is fine.

Reply:
sure. will create new `changelog` with `dch`